sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -220,7 +243,7 @@ internal open class KlaviyoApiRequest(
         .accumulate(METHOD_JSON_KEY, method.name)
         .accumulate(TIME_JSON_KEY, queuedTime)
         .accumulate(UUID_JSON_KEY, uuid)
-        .accumulate(HEADERS_JSON_KEY, JSONObject(headers))
+        .accumulate(HEADERS_JSON_KEY, JSONObject(headers as Map<String, String>))
```

Comment:
```suggestion
        .accumulate(HEADERS_JSON_KEY, JSONObject(headers.toMap()))
```

Reply:
Pretty trivial, but won't `toMap` make a copy of the mutable map, while `as` will just treat it as immutable without copying? Which in this case is sufficient bc we're just writing it into a json object?

Reply:
ah yes, good point, i just like avoiding type casts in kotlin, but this makes sense to keep